spec/image_builder_spec.rb

```diff
@@ -80,16 +80,14 @@
 
       # Build
       expect(builder).to receive(:system)
-        .with({"AYTESTS_FILES_DIR" => files_dir.to_s},
+        .with({"AYTESTS_FILES_DIR" => files_dir.to_s, "AYTESTS_LINUXRC" => "vnc=1",
+               "AYTESTS_PROVIDER" => provider.to_s, "AYTESTS_WEBSERVER_PORT" => "8888"},
```

Comment:
If you don't mind, I'll leave this as it is right now to finish the PBI. I'll come back to work on this right after the sprint.